Carthage/Checkouts/Quick

```diff
@@ -1 +1 @@
-Subproject commit 2f037560be197f0f5ae992512549bc29fabb3818
+Subproject commit dc2b4b75d2f718518acbdef289a72fad77975a8a
```

Comment:
Can you remove the change to the Quick version in this PR? I think that will resolve the conflict and make it so I can merge the PR.